### ISSUE TITLE
HPC implementation for large videos

### ISSUE THREAD

**Issue by NataliaFavila** [bahanonu/ciatah#138]
Dear Biafra,

we are wondering if it would be possible to deploy ciatah for batch processing in a high performance computing cluster?
Is this something that you have done before or have any experience with?

We have large videos and even with temporal (to 5 Hz) and spatial (4x) downsampling, the process is using up our RAM (of around 300 GB).

Any other suggestion is also welcome!

Thank you!
Natalia

**Comment by bahanonu** [comment]
@NataliaFavila 
Yep! It's possible. I've run ciatah on HPCs previously, e.g. on Stanford's HPCs, and it works quite well (e.g. used to analyze 1000+ imaging sessions more quickly or for very large FOV movies).

Depending on your institute's setup, can run parallel processing or cross-node processing. Can either provide more details here or send me an email and we can see how to get it running on your system.

I'll see about updating the repository to add a HPC guide and example scripts as this has been asked previously.
